x-pack/plugins/aiops/public/embeddables/change_point_chart/handle_explicit_input.tsx

```diff
@@ -8,23 +8,22 @@
 import type { CoreStart } from '@kbn/core/public';
 import { toMountPoint } from '@kbn/react-kibana-mount';
 import React from 'react';
-import type { EmbeddableChangePointChartExplicitInput } from './types';
-import type { AiopsAppDependencies } from '..';
-import { AiopsAppContext } from '../hooks/use_aiops_app_context';
-import type { AiopsPluginStartDeps } from '../types';
+import type { ChangePointEmbeddableState } from './types';
+import type { AiopsAppDependencies } from '../..';
+import { AiopsAppContext } from '../../hooks/use_aiops_app_context';
+import type { AiopsPluginStartDeps } from '../../types';
 import { ChangePointChartInitializer } from './change_point_chart_initializer';
-import type { EmbeddableChangePointChartInput } from './embeddable_change_point_chart';
 
 export async function resolveEmbeddableChangePointUserInput(
   coreStart: CoreStart,
   pluginStart: AiopsPluginStartDeps,
-  input?: EmbeddableChangePointChartInput
-): Promise<EmbeddableChangePointChartExplicitInput> {
+  input?: ChangePointEmbeddableState
+): Promise<ChangePointEmbeddableState> {
   const { overlays } = coreStart;
 
   return new Promise(async (resolve, reject) => {
     try {
-      const modalSession = overlays.openModal(
+      const flyoutSession = overlays.openFlyout(
```

Comment:
nit: If you're opening a flyout, you can let the Dashboard know. This allows the Dashboard to blur other panels, disable buttons on the top of the page, and close the flyout if the user navigates away.

You can do this with the following code:

``` typescript
if (apiHasParentApi(api) && tracksOverlays(api.parentApi)) {
  api.parentApi.openOverlay(overlay, { focusedPanelId: api.uuid });
}
```

Reply:
Added in https://github.com/elastic/kibana/pull/182246/commits/585cfae1c7ea36cc75408f52e0c8e6349378b409